examples/hex_neighbors_radius_chunks.rs

```diff
@@ -312,24 +345,47 @@ fn swap_map_type(
 #[derive(Component)]
 struct TileLabel(Entity);
 
+const DEFAULT_FONT_SIZE: f32 = 14.0;
+const BIGGER_FONT_SIZE: f32 = 16.0;
```

Comment:
nit: `DEFAULT_` feels *slightly* confusing due to the default on `TextFont` being `20`. But I would personally ditch the new "increased font size on hover" effect. There just isn't much room in the tiles to increase the size.

Reply:
Removed the `BIGGER_FONT_SIZE`.